.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+
+---
+
```

Comment:
Is it worth prompting the user for installation information (OS, version for RMG-Py, RMG-database, ARC) like bug requests do for RMG-Py? Admittedly, I’ve rarely supplied this information, so this is up to you as it’s just a question.

Reply:
I think we don't need to prompt, for the reason you specified (we can always ask if it's relevant and not supplied)